lib/DirectoryWatcher.js

```diff
@@ -46,23 +46,13 @@ function DirectoryWatcher(directoryPath, options) {
 	this.path = directoryPath;
 	this.files = {};
 	this.directories = {};
-	this.watcher = chokidar.watch(directoryPath, {
-		ignoreInitial: true,
-		persistent: true,
-		followSymlinks: false,
-		depth: 0,
-		atomic: false,
-		alwaysStat: true,
-		ignorePermissionErrors: true,
+	this.watcher = sane(directoryPath, {
```

Comment:
You're completely right and I'm kinda embarrassed that I didn't figure that out earlier.
I wrote a cheap one-line filter for this and it worked.  I'll see if I can get a more efficient one hammered out by tomorrow.